src/styles/shared/components/buttons/_base.scss

```diff
@@ -0,0 +1,118 @@
+@use "sass:map";
+
+@use "@hypothesis/frontend-shared/styles/mixins/focus";
+
+// Set colors for a button
+@mixin _colors($colormap) {
```

Comment:
`_colors`, `_hover-state`, `_active-state` are unchanged except for private-namespace prefix (`_`)